src/clusters.tsx

```diff
@@ -45,13 +51,19 @@ export class DaskClusterManager extends Widget {
   /**
    * Create a new Dask cluster manager.
    */
+  status = 'ready';
+  registry: CommandRegistry;
+  launchClusterId: string;
```

Comment:
I think these can be private, and moved down with the rest of the private attributes.

Reply:
Ok sounds good.